### PR TITLE
[UAR-1273] Make individual trustee DOB uneditable when reviewing

### PR DESCRIPTION
**Pull request by corybudd-ch**
### JIRA link
https://companieshouse.atlassian.net/browse/UAR-1273

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: src/controllers/update/update.manage.trusts.tell.us.about.the.individual.controller.ts [hunk 1]
```diff
@@ -17,11 +17,12 @@ import { getTrustInReview, getTrustee, getTrusteeIndex } from '../../utils/updat
...
 import { FormattedValidationErrors, formatValidationError } from '../../middleware/validation.middleware';
 import { TrusteeType } from '../../model/trustee.type.model';
-import { IndividualTrustee, Trust } from '../../model/trust.model';
+import { IndividualTrustee, Trust, TrustIndividual } from '../../model/trust.model';
 import { RoleWithinTrustType } from '../../model/role.within.trust.type.model';
 import { IndividualTrusteesFormCommon } from '../../model/trust.page.model';
 
-const getPageProperties = (trust, formData, errors?: FormattedValidationErrors) => {
+const getPageProperties = (trust, formData, trustee: TrustIndividual, errors?: FormattedValidationErrors) => {
+  // if DOB present & CH references set uneditaleDOB to true
```
Comment (AndreKramer7654) [inline src/controllers/update/update.manage.trusts.tell.us.about.the.individual.controller.ts:25]: no real need for this comment

File: src/controllers/update/update.manage.trusts.tell.us.about.the.individual.controller.ts [hunk 2]
```diff
@@ -67,9 +69,11 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
     const formData: IndividualTrusteesFormCommon = req.body as IndividualTrusteesFormCommon;
 
     if (!errorList.isEmpty()) {
+      const trustee = getTrustee(trust, trusteeId, TrusteeType.INDIVIDUAL) as IndividualTrustee;
```
Comment (AndreKramer7654) [inline src/controllers/update/update.manage.trusts.tell.us.about.the.individual.controller.ts:72]: "as" is the developer asserting that the typing is as stated.

File: views/includes/page-templates/trust-individual-beneficial-owner.html [hunk 3]
```diff
@@ -58,18 +58,23 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ pageTitle }}</h1>
...
-          'Day': formData.dateOfBirthDay,
-          'Month': formData.dateOfBirthMonth,
-          'Year': formData.dateOfBirthYear
-        },
-        label: 'What is their date of birth?',
-        error: errors.dateOfBirth if errors
-      } %}
-
-      {% include "includes/inputs/date-input.html" %}
+      {% if uneditableDOB %}
```
Comment (AndreKramer7654) [inline views/includes/page-templates/trust-individual-beneficial-owner.html:61]: Would be better in a template to include than inline here

File: src/controllers/update/update.manage.trusts.tell.us.about.the.individual.controller.ts (hunk 1 above)
Comment (corybudd-ch) [reply src/controllers/update/update.manage.trusts.tell.us.about.the.individual.controller.ts:25]: woops, that was a leftover thought :D

File: views/includes/page-templates/trust-individual-beneficial-owner.html (hunk 3 above)
Comment (corybudd-ch) [reply views/includes/page-templates/trust-individual-beneficial-owner.html:61]: No, just wrapped existing HTML with the if statement is the only change here.

File: src/controllers/update/update.manage.trusts.tell.us.about.the.individual.controller.ts (hunk 2 above)
Comment (corybudd-ch) [reply src/controllers/update/update.manage.trusts.tell.us.about.the.individual.controller.ts:72]: Correct

**Review by bbell-companieshouse — APPROVED** [review]

**Review by cdrainCH — APPROVED** [review]

**Review by keelan-scott — APPROVED** [review]

**Review by stephanie-ch — APPROVED** [review]

**Pull request merged**